tests/search/casedmatch/casedmatch.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright Vespa.ai. All rights reserved.
+
+require 'indexed_streaming_search_test'
```

Comment:
this won't work on streaming (yet), did you intend to have a known-to-fail test?